src/session/content/message_row/text.rs

```diff
@@ -77,7 +77,7 @@ impl MessageText {
                     ChatType::BasicGroup(_) | ChatType::Supergroup(_)
                 )
             } else {
-                false
+                matches!(message.sender(), MessageSender::Chat(_))
```

Comment:
I think it's fine if we also remove the negation in the same course. That makes the whole thing easier to read.

Like:
```rust
let show_sender = {
    if message.is_outgoing() {
        matches!(message.sender(), MessageSender::Chat(_))
    } else {
        matches!(
            message.chat().type_(),
            ChatType::BasicGroup(_) | ChatType::Supergroup(_)
        )
    }
};
```

Reply:
Right, agreed.